README.md

```diff
@@ -752,46 +752,42 @@ var signIn = new OktaSignIn(config);
 
 ### OIDC Applications
 
-For OIDC applications, you need to set the `clientId` and `redirectUri` (in addition to `baseUrl`).
+For OIDC applications, you need to set the `clientId` and `redirectUri`. If `issuer` is not set, it will be inferred from `baseUrl`.
 
 ```javascript
 var config = {
-  baseUrl: 'https://{yourOktaDomain}',
+  baseUrl: 'https://{yourOktaDomain}', // issuer will be https://{yourOktaDomain}/oauth2/default
   clientId: '{{clientId of your OIDC app}}',
   redirectUri: '{{redirectUri configured in OIDC app}}'
 }
 ```
 
-By default, the issuer ([Authorization Server][]) will be set to the default [Custom Authorization Server][]: 
+By default, the issuer ([Authorization Server][]) will be set to the default [Custom Authorization Server][].
 
 ```javascript
 // default issuer
 config.baseUrl + '/oauth2/default'
 ```
 
-A different [Custom Authorization Server][] can be specified by setting the `issuer` in the `authParams` section:
+A different [Custom Authorization Server][] can be specified by setting the `issuer` explicitly:
 
 ```javascript
 var config = {
-  baseUrl: 'https://{yourOktaDomain}',
+  issuer: 'https://{yourOktaDomain}/oauth2/custom',
```

Comment:
removing baseUrl entirely ? 

can't be `baseUrl == issuer` ?

Reply:
Yes, it can be removed. I was in the process of updating the README, but decided to hold off as this will be merged to master before the change is pushed. There can be a README update after this version is publicly released